docs/ecmarkup.js

```diff
@@ -119,7 +119,11 @@ Search.prototype.search = function (searchString) {
     
     for (var i = 0; i < this.biblio.length; i++) {
       var entry = this.biblio[i];
-  
+      if (!entry.key) {
```

Comment:
Is there a bug for this on https://github.com/bterlson/ecmarkup?

Reply:
No. Why is this a bug? I'm not sure how ecmarkup generates this output, feel free to file an issue if you think this is incorrect.